canonical-web-and-design/maas-ui
# Add delete domain button to domail detail header

## Done

- Add delete domain button to domain detail header

## QA
Go to http://0.0.0.0:8400/MAAS/r/domains
pick a domain with records, try to delete it and see that you can't
go back, pick a domain without records (or create a new one) and delete it

Fixes: https://github.com/canonical-web-and-design/app-squad/issues/45

![image](https://user-images.githubusercontent.com/11927929/122075322-477fad80-cdfa-11eb-8985-ea2e77505697.png)

## Note 
This is branched of off the domains detail header branch so, don't merge this one first!
